Packs/CommonScripts/ReleaseNotes/1_4_41.md

```diff
@@ -0,0 +1,4 @@
+
+#### Scripts
+##### URLSSLVerification
+- Added an argument that will allow the user to choose whether in case that the URL starts with HTTP and not HTTPS, to make the URL Suspicious.
```

Comment:
```suggestion
- Added the argument *set_http_as_suspicious* that will allow the user to choose whether to set the URL as Suspicious in case that the URL starts with HTTP and not HTTPS.
```